### PR TITLE
Don't use getenv(), use $_SERVER instead

### PR DESCRIPTION
**Pull request by yceruto**
See https://github.com/symfony/recipes/pull/156

### CODE REVIEW

**Comment by javiereguiluz** [comment]
Thanks @yceruto.

File: public/index.php [hunk 1]
```diff
@@ -17,11 +17,11 @@
 require __DIR__.'/../vendor/autoload.php';
 
 // The check is to ensure we don't use .env in production
-if (!getenv('APP_ENV')) {
+if (!isset($_SERVER['APP_ENV'])) {
     (new Dotenv())->load(__DIR__.'/../.env');
 }
 
-if (getenv('APP_DEBUG')) {
+if ($_SERVER['APP_DEBUG'] ?? true) {
```
Comment (stof) [inline public/index.php]: this looks broken, as setting it to `'0'` will still be a truthy string.

File: public/index.php [hunk 2]
```diff
@@ -32,7 +32,7 @@
 
 // Request::setTrustedProxies(['0.0.0.0/0'], Request::HEADER_FORWARDED);
 
-$kernel = new Kernel(getenv('APP_ENV'), getenv('APP_DEBUG'));
+$kernel = new Kernel($_SERVER['APP_ENV'] ?? 'dev', $_SERVER['APP_DEBUG'] ?? true);
```
Comment (stof) [inline public/index.php]: this is weird too. The second argument should be a boolean. The string `'0'` (as supported by the console) won't be false.

File: public/index.php (hunk 1 above)
Comment (yceruto) [reply public/index.php]: and `'0' == false` is true

File: public/index.php (hunk 2 above)
Comment (yceruto) [reply public/index.php]: and `(bool) '0'` is `false` 

https://github.com/symfony/symfony/blob/edb8c33f35b981b72adbe23798753703edfe3819/src/Symfony/Component/HttpKernel/Kernel.php#L83

**Pull request merged**